test/dev/basic.dynamic.test.js

```diff
@@ -62,6 +62,14 @@ describe('basic ssr', () => {
     expect(fetchCount).toBeGreaterThan(0)
   })
 
+  test('dynamic publicPath is used in ssr html', async () => {
+    const { html } = await nuxt.server.renderRoute('/', {
+      resources: resourceLoader
+    })
+
+    expect(html).toContain('<img src="https://cdn.nuxtjs.org/img')
```

Comment:
currently producing `https:/cdn.nuxtjs.org/img`

Reply:
fyi: issue is `urlJoin` - which doesn't handle full urls - would have caused bugs elsewhere that were being normalised by browser

Reply:
Thank you